src/templates/ReportForm.tsx

```diff
@@ -66,7 +68,7 @@ const ReportForm = ({ data, location }) => {
   // state
   const [state, setState] = useState<ReportState>(initialState);
   const [attributedTo, setAttributedTo] = useState<string>("");
-  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
```

Comment:
This was actually set to true by default for a purpose. We want to show a loading state until the finding details are fetched when we're editing findings. This same page is used for submitting and editing findings. If it's set to false by default, then we'll see the form populate with data for a new finding and then abruptly update when the finding details are fetched.

Reply:
Good catch, I changed that for testing something, I removed all my testing code, but I missed this! I will change it back now!